### PR TITLE
deprecate goerli (+ zksync, base, arbitrum, optimism)

### PR DESCRIPTION
**Pull request by CoveMB**
deprecate goerli (+ zksync, base, arbitrum, optimism)

### CODE REVIEW

File: src/cmd/deploy.ts [hunk 1]
```diff
@@ -775,7 +775,6 @@ export default class DefenderDeploy {
             notificationChannels: match.notifyConfig?.notifications.map(
               (n: DefenderNotificationReference) => n.notificationId,
             ),
-            severityLevel: match.notifyConfig?.severityLevel,
```
Comment (CoveMB) [inline src/cmd/deploy.ts:778]: @shahnami Had a type issue here does it make sense to remove it?

Comment (shahnami) [reply src/cmd/deploy.ts:778]: No, you'll need to yarn link the latest SDK main branch since the SDK hasn't been published yet.

**Review by shahnami — APPROVED** [review]
Apart from removing severityLevel, lgtm!

**Review by MCarlomagno — APPROVED** [review]

File: src/types/docs/definitions-definitions-publicnetwork.md [hunk 2]
```diff
@@ -43,10 +41,8 @@
 | `"hedera"`              |             |
 | `"hederatest"`          |             |
 | `"zksync"`              |             |
-| `"zksync-goerli"`       |             |
 | `"zksync-sepolia"`      |             |
 | `"base"`                |             |
-| `"base-goerli"`         |             |
 | `"base-sepolia"`        |             |
 | `"linea-goerli"`        |             |
```
Comment (tirumerla) [inline src/types/docs/definitions-definitions-publicnetwork.md:47]: Remove this as well

File: src/types/docs/definitions-definitions-publicnetwork.md [hunk 3]
```diff
@@ -11,7 +11,6 @@
 | `"mainnet"`             |             |
 | `"sepolia"`             |             |
 | `"holesky"`             |             |
-| `"goerli"`              |             |
 | `"xdai"`                |             |
 | `"sokol"`               |             |
```
Comment (tirumerla) [inline src/types/docs/definitions-definitions-publicnetwork.md:15]: sokol is also deprecated

**Review by tirumerla — COMMENTED** [review]
Added few comments lets remove linea goerli and sokol as well

**Review by tirumerla — APPROVED** [review]
lgtm

**Pull request merged**